ext/node/ops/buffer.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+
+use deno_core::op2;
+use deno_core::v8;
+
+#[op2(fast)]
+pub fn op_is_ascii(buf: v8::Local<v8::ArrayBufferView>) -> bool {
```

Comment:
You can use the `#[buffer]` attribute here and the op2 macro will do the conversion

```rust
pub fn op_is_ascii(#[buffer] buf: &[u8]) -> bool {
```